resources/assets/less/bem/beatmap-discussion-nomination.less

```diff
@@ -0,0 +1,81 @@
+/**
+ *    Copyright 2015 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+.beatmap-discussion-nomination {
+  margin-top: 10px;
+
+  &__disqualification-banner {
+    width: 100%;
+    min-height: 50px;
+    padding: 10px;
+    font-size: 14px;
+    border-left: 5px solid @pink;
+  }
+
+  &__disqualification-time {
+    display: block;
+    width: 100%;
+    text-align: right;
+    font-size: 10px;
+    margin-top: 5px;
+  }
+
+  &__message-area, &__message-area-qualified {
+    width: 100%;
+    min-height: 50px;
+    padding: 10px;
+    font-size: 12px;
+    text-align: center;
+    background-color: @yellow-lighter;
+
+    display: flex;
+    flex: 1 0 auto;
+    flex-direction: column;
+    @media @desktop {
+      flex-direction: row;
+    }
+  }
+
+  &__message-area-qualified {
+    background-color: @green;
+    color: white;
+  }
+
+  &__message-text {
+    top: 6px;
+    font-size: 14px;
+    padding-bottom: 15px;
+    @media @desktop {
+      padding-bottom: 15px;
+      flex-basis: 85%;
```

Comment:
looks weird without the button. Just put a `flex: none` for button and `flex: 1` here